buildMacDist.sh

```diff
@@ -1,117 +1,9 @@
 #!/bin/sh
 
-# remove old build
-/bin/rm -rf build
-/bin/rm -rf dist
-/bin/rm -rf dist_dmg
-
-# set the build date in version.py
-python3.4 buildVersion.py
-
+# Rebuild messages.pot internationalization file
```

Comment:
At a minimum `set -x` should provide improved logging in the github workflow runs. `set -xeu` is a good practice for any of these workflow scripts.